tests/cases/onboarders_care_platform/case_onboarders_care_platform_account.yaml

```diff
@@ -471,4 +471,37 @@ TestOnboarderClickAndViewFacilitiesDevice:
       Asserts:
       - Type: contain
         Var: STATUS_CLASS 
-        Value: 'active'
\ No newline at end of file
+        Value: 'active'
+
+# CXTB-185 Can Click and View Facility Device Record from Listing
+TestOnboarderCanClickAndViewFacilitiesDevice:
+  Environment: Settings
+  Roles:
+    - Role: desktopChrome
+      App: "desktop"
+  Actions:
+    # Login-in onboarder Care Partner
+    - Type: case
+      Value: OnboarderCarePartnerLogin
+    # Click in facilities button
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility.facilities_button$
+    # Click on the first listing record
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.first_senior_lastname$
+    # Click on record button and check the Devices
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant_details.records_button$
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility_Record.info_title$
+    # Click on record button and check the Devices
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_facility_Record.devices_button$
+    - Type: get_attribute
```

Comment:
After getting the attribute there should be an assertion

Reply:
fixed